WooCommerce/src/main/kotlin/com/woocommerce/android/ui/products/ai/preview/AiProductPreviewScreen.kt

```diff
@@ -0,0 +1,419 @@
+        Spacer(Modifier)
+        LoadingSkeleton(
+            modifier = Modifier
+                .fillMaxWidth()
+                .then(sectionsBorder)
+        )
+    }
+}
+
+@Composable
+private fun ErrorDialog(
+    onRetryClick: () -> Unit,
+    onDismissClick: () -> Unit
+) {
+    AlertDialog(
+        onDismissRequest = {},
+        properties = DialogProperties(dismissOnClickOutside = false, dismissOnBackPress = false),
+        text = {
+            Text(text = stringResource(id = R.string.product_creation_ai_generation_failure_message))
+        },
+        confirmButton = {
+            WCTextButton(onClick = onRetryClick) {
+                Text(stringResource(id = R.string.retry))
+            }
+        },
+        dismissButton = {
+            WCTextButton(onClick = onDismissClick) {
+                Text(stringResource(id = R.string.dismiss))
+            }
+        }
+    )
+}
+
+@Composable
+@Preview
+@PreviewLightDark
+private fun ProductPreviewLoadingPreview() {
+    WooThemeWithBackground {
+        AiProductPreviewScreen(
+            state = AiProductPreviewViewModel.State.Loading,
+            onFeedbackReceived = {},
+            onBackButtonClick = {}
+        )
+    }
+}
+
+@Composable
+@Preview
+@PreviewLightDark
+private fun ProductPreviewContentPreview() {
+    WooThemeWithBackground {
+        AiProductPreviewScreen(
+            state = AiProductPreviewViewModel.State.Success(
+                product = AIProductModel.buildDefault(
+                    name = "Soft Black Tee: Elevate Your Everyday Style",
+                    description = "Introducing our USA-Made Classic Organic Cotton Tee—a staple piece designed for" +
+                        " everyday comfort and sustainability. Crafted with care from organic cotton, this tee is not" +
+                        " just soft on your skin but gentle on the environment."
+                ),
+                propertyGroups = listOf(
+                    listOf(
+                        AiProductPreviewViewModel.ProductPropertyCard(
+                            icon = R.drawable.ic_gridicons_product,
+                            title = R.string.product_type,
+                            content = "Simple Product"
+                        )
+                    ),
+                    listOf(
+                        AiProductPreviewViewModel.ProductPropertyCard(
+                            icon = R.drawable.ic_gridicons_money,
+                            title = R.string.product_price,
+                            content = "Regular price: $45.00"
+                        ),
+                        AiProductPreviewViewModel.ProductPropertyCard(
+                            icon = R.drawable.ic_gridicons_list_checkmark,
+                            title = R.string.product_inventory,
+                            content = "In stock"
+                        )
+                    )
+                )
+            ),
+            onFeedbackReceived = {},
```

Comment:
Minor but this is missing `selectedVariant` in order for the code to compile. 

```suggestion
                    )
                ),
                selectedVariant = 0,
            ),
            onFeedbackReceived = {},
```

Reply:
Sorry about this, I made some last minute changes, and this was a left over.